qt/scientific_interfaces/ISISReflectometry/test/Reduction/LookupTableTest.h

```diff
@@ -291,7 +291,7 @@ class LookupTableTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(3, table.rows().size());
 
     auto newLookupRow = ModelCreationHelper::makeLookupRow(angle, boost::regex("A.*"));
-    auto const signalProcessingInstructions = "5678";
+    auto const signalProcessingInstructions = std::string{"5678"};
```

Comment:
This isn't worth holding this up but just a note that you can create a `std::string` from a literal using the `s` suffix in the `literals` namespace - https://en.cppreference.com/w/cpp/string/basic_string/operator%22%22s so that it's slightly less cumbersome to type.